portfolio/src/main/webapp/script.js

```diff
@@ -74,12 +74,29 @@ function showCaption(element) {
 }
 
 /*
- * Fetch data from the data servlet to display on main page.
+ * Fetch comments data from the data servlet to display on main page.
  */
-function getServerMessage() {
+function getComments() {
   fetch('/data')
-  .then(response => response.text())
-  .then((message) => {
-    document.getElementById('message-container').innerText = message;
+  .then(response => response.json())
+  .then((comments) => {
+    console.log(comments);
+    // comments is an array of json objects
+    const commentsElement = document.getElementById('comments-container');
+    commentsElement.innerHTML = '';
+    for (i in comments) {
+      commentsElement.appendChild(
+        createElement('By: ' + comments[i].name + '\n Message: ' + comments[i].message));
+    }
```

Comment:
I believe this is moreso due to how they've told y'all to do it, but this looks to be erasing all the previous comments and rewriting them all. Ideally we'd like to just append the new ones. Not telling you to try to fix it now, but if you want, we can add a TODO here to try to revisit this towards the end of the week to try to do that if there is time

Reply:
Okay, I can try to revisit that later!